### PR TITLE
Update microsoft-365-guest-settings.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as Sharing is under Policies in the new sharepoint admin center

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8098

File: microsoft-365/solutions/microsoft-365-guest-settings.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ These settings affect all of the sites in the organization. They do not affect M
 
 Because OneDrive is a hierarchy of sites within SharePoint, the organization-level sharing settings directly affect OneDrive just as they do other SharePoint sites.
 
-**Navigation:** SharePoint admin center > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
+**Navigation:** SharePoint admin center > Policies > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-guest-settings.md]:
```suggestion
**Navigation:** SharePoint admin center > **Policies** > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
```

File: microsoft-365/solutions/microsoft-365-guest-settings.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ Because OneDrive is a hierarchy of sites within SharePoint, the organization-lev
 
 ### SharePoint and OneDrive advanced sharing settings
 
-**Navigation:** SharePoint admin center > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
+**Navigation:** SharePoint admin center > Policies > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-guest-settings.md]:
```suggestion
**Navigation:** SharePoint admin center > **Policies** > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
```

File: microsoft-365/solutions/microsoft-365-guest-settings.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ Because OneDrive is a hierarchy of sites within SharePoint, the organization-lev
 
 When files and folders are shared in SharePoint and OneDrive, sharing recipients are sent a link with permissions to the file or folder rather than being granted direct access to the file or folder themselves. Several types of links are available, and you can choose the default link type presented to users when they share a file or folder. You can also set permissions and expiration options for *Anyone* links.
 
-**Navigation:** SharePoint admin center > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
+**Navigation:** SharePoint admin center > Policies > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-guest-settings.md]:
```suggestion
**Navigation:** SharePoint admin center > **Policies** > <a href="https://go.microsoft.com/fwlink/?linkid=2185222" target="_blank">**Sharing**</a>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**